python/pyarrow/io.pxi

```diff
@@ -868,6 +868,15 @@ def foreign_buffer(address, size, base):
     return pyarrow_wrap_buffer(buf)
 
 
+def jvm_buffer(arrowbuf):
+    """
+    Construct an Arrow buffer from io.netty.buffer.ArrowBuf
```

Comment:
Just for my enlightenment, this works only with JPype or with any Java/Python bridge (are there any others?)?

Reply:
It may also work with https://www.py4j.org/ but I have not tested this yet

Reply:
Thinking a bit more, it might not work with Py4J as the JVM memory addresses returned there are probably in another process.